octoprint_mrbeam/templates/wizard/wizard_analytics.jinja2

```diff
@@ -6,119 +6,62 @@
 {# Wording changes must be done in both templates: wizard_analytics.jinja and analytics_settings.jinja !! #}
 {# !!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!! #}
 
-
+<h3>{{ _("Analytics") }}</h3>
+{% set docs_locale = 'en' %}
```

Comment:
do we still need this variable? if not , please remove it

Reply:
Yes, we still need it for the "MrBeamAnalytics-PrivacyPolicy_%s"

Reply:
ok